routes/project.js

```diff
@@ -31,7 +33,16 @@ module.exports = {
  * ]
  */
 function getProjects(req, res, next) {
-  Project.findAll()
+  let search = {};
+
+  //FIXME: this is probably not the best way to implement this
+  // since a search for a name should always return a maximum of 1 item
+  if (req.query.name) {
+    search.where = {
+      name: req.query.name
```

Comment:
What if the project name has a space in it? Does this get interpreted in `req.query.name` as something like `My%20Project`?  If so, here's an example of how I implemented handling around URL strings:

https://github.com/osmlab/to-fix-backend/blob/10e3e31ae2a72f8928a559b9bc6495d5a804e4a2/routes/tag.js#L51-L54

Reply:
So I thought of this, but the test that I have does search for a project with a space in its name, and it works fine.

From my experience with frameworks, they should always handle query string decoding for you transparently - so when making the query, of course, you need to `encodeURIComponent` your query strings, but when reading it out in express, express should just handle this for you internally, and as per the passing tests we have, that works :-) - fine to leave as is?

Reply:
👍  Sounds good!